packages/@n8n/api-types/src/schemas/user.schema.ts

```diff
@@ -22,12 +22,15 @@ export const userProjectSchema = z.object({
 	name: z.string(),
 });
 
-export const userListItemSchema = z.object({
+export const userMemberSchema = z.object({
```

Comment:
The naming userMemberSchema and userAdminSchema feel a bit misleading, because when I saw them, I thought it would represent the information of a member/admin user, the the user information that a member/admin **can see**.
How about a classic like 'userBaseSchema' and 'userDetailSchema' ?